Automattic/wp-calypso
# Help Center: Manage notifications in icon and minimized

Fixes https://github.com/Automattic/dotcom-forge/issues/9341

This PR calculates the unread count on load and based on that the appropriate help center icon is displayed.
Also when the help center is minimized the unread count is displayed and the header is styled differently in this state.

| Before  | After |
| ------------- | ------------- |
| <img width="883" alt="Screenshot 2024-10-29 at 13 02 18" src="https://github.com/user-attachments/assets/ef6da694-60ab-4302-ba77-0be1d7663bb6">  |  <img width="887" alt="Screenshot 2024-10-29 at 13 02 04" src="https://github.com/user-attachments/assets/88b01614-126b-428d-b273-194d6345a37f"> |
| <img width="889" alt="Screenshot 2024-10-29 at 13 02 32" src="https://github.com/user-attachments/assets/ca86505c-9911-451f-8495-fa7db7b8cb71"> | <img width="386" alt="Screenshot 2024-10-29 at 13 02 50" src="https://github.com/user-attachments/assets/22dfdb33-08b2-410c-b69f-5514d5726a88">|

## Proposed Changes

* Utilize `unreadCount` and set it based on the unread conversations
* Style the help center differently based on this
* Style the help center icon differently based on this

## Testing Instructions
- Test both in calypso and wp-admin
- Create conversations with unread messages
- Check if the appropriate icon is displayed based on this
- Check the header when the help center is minimized. It should be styled differently based on the figma



*
